src/console/Command.cpp

```diff
@@ -0,0 +1,103 @@
+#include "console/Command.hpp"
+#include "storm/Error.hpp"
+
+// Extern
+TSHashTable<CONSOLECOMMAND, HASHKEY_STRI> g_consoleCommandHash;
+char g_commandHistory[HISTORY_DEPTH][CMD_BUFFER_SIZE];
+uint32_t g_commandHistoryIndex;
+
+
+void ConsoleCommandDestroy() {
+    g_consoleCommandHash.Clear();
+}
+
+char* ConsoleCommandHistory(uint32_t index) {
+    // Return a pointer to the buffer at the specified index
+    return g_commandHistory[((g_commandHistoryIndex + (HISTORY_DEPTH - 1) - index) & (HISTORY_DEPTH - 1))];
+}
+
+uint32_t ConsoleCommandHistoryDepth() {
+    return HISTORY_DEPTH;
+}
+
+void ConsoleCommandInitialize() {
+    ConsoleCommandRegister("help", ConsoleCommand_Help, CONSOLE, "Provides help information about a command.");
+}
+
+int32_t ConsoleCommandRegister(const char* command,
+                               int32_t (*handler)(const char*, const char*),
+                               CATEGORY category,
+                               const char* helpText) {
```

Comment:
Let's keep these declarations on a single line. Is `.clang-format` suggesting this? We can tweak the rules a bit, if so.

Reply:
This can be suggested by `.clang-format` depending on the code style guidelines, because it makes it easier to read the function signature on smaller screens (think vertical orientation or windowed mode).

Reply:
I realize it's wrapping from column length requirements, but I'd prefer to not have it wrap until I better think through the rules for continuation indents and etc.